numba_rvsdg/rendering/rendering.py

```diff
@@ -4,28 +4,28 @@
     RegionBlock,
     PythonBytecodeBlock,
     SyntheticAssignment,
-    SyntheticExitingLatch,
-    SyntheticExitBranch,
     SyntheticBranch,
     SyntheticBlock,
-    SyntheticHead,
-    SyntheticExit,
 )
 from numba_rvsdg.core.datastructures.scfg import SCFG
 from numba_rvsdg.core.datastructures.byte_flow import ByteFlow
 import dis
 from typing import Dict
 
-class BaseRenderer(object):
+
+class BaseRenderer:
     """Base Renderer class.
 
     This is the base class for all types of graph renderers. It defines two
-    methods `render_block` and `render_edges` that define how the blocks and edges
-    of the graph are rendered respectively.
+    methods `render_block` and `render_edges` that define how the blocks and
+    edges of the graph are rendered respectively.
     """
 
-    def render_block(self, digraph: "Digraph", name: str, block: BasicBlock):
-        """Function that defines how the BasicBlcoks in a graph should be rendered.
+    def render_block(
+        self, digraph: "Digraph", name: str, block: BasicBlock  # noqa
```

Comment:
The `# noqa` here is because it can't find the `Digraph` for the annotation?

Reply:
Yes, otherwise flake8 complains about an undefined `Digraph` object